### DISCUSSION TITLE
Button for underlining text

### DISCUSSION THREAD

**Discussion by erichjsonfosse** [formfcw/directus-extension-flexible-editor#33] [Feature Request]
Hi,

First of all, thanks for an awesome extension! It has really come in handy :100: 

It does seem like it's missing the button for underlining text, tho. Is there a way for me to contribute to having it added?

**Comment by formfcw** [comment]
Thank you for your interest!

Feel free to contribute. Here is a comment about adding extensions: https://github.com/formfcw/directus-extension-flexible-editor/discussions/27#discussioncomment-7241970
